### ISSUE TITLE
Synchronize figma artifacts to github repo.

### ISSUE THREAD

**Issue by jan-gogo** [Flank/flank#1625]
**Author the user story for this feature**

As a designer, I want to have auto-synchronization of Figma artifacts with GitHub repo so I can take all advantages of the git version control.

**Describe the solution**

Sync the raw native figma files to a github repo, similar to what you'd get if you manually pressed the download button.

This feature is not directly related to the Flank itself, so it can be added as standalone project in the Flank github but not directly into core flank repo.

* Prepare SDD
* Make and abstraction for figma API (only related to this issue scope)
* Create integration with github (or just git, if is enough)
* Choose best way to trigger the synchronization

**References**

* https://github.com/marcomontalbano/figma-export (quite similar project, check out If can be useful)

**Comment by piotradamczyk5** [comment]
# Project name

> Summary of the proposed change

# References 

- https://github.com/marcomontalbano/figma-export 

# Motivation

> What problem is this project solving?

# Goals

> Identify success metrics and measurable goals.

# Non-Goals [optional]

> Identify what's not in scope.

# Design

> Explain and diagram the technical design
>
> Identify risks and edge cases

# API

> What will the proposed API look like?
  
# Results

> What was the outcome of the project?

# Dependencies

> What is the project blocked on?

> What will be impacted by the project?

# Testing

> How will the project be tested?

# Alternatives Considered [optional]

> Summarize alternative designs (pros & cons)

# Timeline [optional for regular tigers]

> Document milestones and deadlines.

DONE:

  -

NEXT:

  -

**Comment by jan-gogo** [comment]
I am not sure about goal nr 1. IMO it is the end-user responsibility, not ours.
